test/performance/dataplane-probe/sla.go

```diff
@@ -147,8 +148,9 @@ var (
 				Method: "GET",
 				URL:    "http://activator-with-cc.default.svc.cluster.local?sleep=100",
 			},
-			stat:      "ac",
-			estat:     "be",
+			stat:  "ac",
+			estat: "be",
+			// We use the same threshold analyzer, since we want Breaker to exert minimal latency impact.
```

Comment:
Is it the breaker both places?  Throttler?

Reply:
Throttler is breaker manager in activator.

Reply:
I'd actually say "Throttler/Breaker". The throttler's logic might be non-trivial as well and add latency (which we expect not to be there).

Reply:
Done!